ui-pf4/src/main/webapp/package.json

```diff
@@ -5,6 +5,7 @@
   "proxy": "http://localhost:8080",
   "homepage": "./",
   "dependencies": {
+    "@adobe/css-tools": "^4.3.3",
```

Comment:
As this is not a direct dependency for the UI. It is better to set this line under the `resolutions` section of the package.json

Reply:
this line has been moved as suggested